docs/courses/robotics/3.3-liquid-handlers.md

```diff
@@ -7,11 +7,59 @@
 
 ## 🔰 Tutorial
 
-In this module, you will perform liquid transfer between containers with an automated liquid handler (Science Jubilee and Opentrons OT-2).
+Welcome to the fascinating world of automated liquid handling! 🌊 In this module, you'll learn how to perform liquid transfers between containers using two cutting-edge systems: the Science Jubilee and the Opentrons OT-2. 
+
+### The Liquid Handling Superstars 🌟
+
+1. **Science Jubilee** 🔬
+   Imagine a Swiss Army knife for scientists - that's Science Jubilee! It's a versatile, open-source platform that can handle various lab tasks, including precise liquid transfers.
+
+   Fun fact: Science Jubilee is like a scientific chameleon, adapting to different research needs! 🦎
+
+2. **Opentrons OT-2** 🤖
+   Meet the friendly neighborhood robot scientist! The OT-2 is a popular, user-friendly liquid handling robot that's making waves in labs worldwide.
+
+   Did you know? The OT-2 can pipette volumes as small as 1 microliter - that's about the size of a grain of salt! 🧂
+
+### Why Automated Liquid Handling Rocks! 🎸
+
+1. **Precision** 🎯
+   These robots can measure and dispense liquids with incredible accuracy, reducing human error.
+
+2. **Consistency** 🔄
+   They perform the same task identically every time, ensuring reproducible results.
+
+3. **Efficiency** ⏱️
+   They can work 24/7 without getting tired, speeding up experiments.
+
+4. **Safety** 🛡️
+   They can handle hazardous materials, keeping human researchers safe.
+
+### Real-World Applications 🌍
+
+1. **Drug Discovery** 💊
+   Automated liquid handlers are crucial in high-throughput screening of potential drug compounds.
+
+2. **Genomics Research** 🧬
+   They're indispensable for tasks like DNA extraction and PCR setup.
```

Comment:
Probably better to use softer language like "well-suited for" or "often used for", etc.